cmake/FindMKL.cmake

```diff
@@ -1,356 +1,302 @@
-#
-# Distributed Linear Algebra with Future (DLAF)
-#
-# Copyright (c) 2018-2019, ETH Zurich
-# All rights reserved.
-#
-# Please, refer to the LICENSE file in the root directory.
-# SPDX-License-Identifier: BSD-3-Clause
-#
+#[=======================================================================[.rst:
```

Comment:
What's this line for?

Reply:
That's the usual way of writing documentation in Find modules : https://github.com/Kitware/CMake/blob/master/Modules/FindOpenMP.cmake